docs/gen_man.d

```diff
@@ -43,10 +43,22 @@ The actual linking is done by running \fBgcc\fR.
 This ensures compatibility with modules compiled with
 \fBgcc\fR.
 .SH FILES
+.TP
 .I /etc/dmd.conf
-dmd will look for the initialization file
+\fBdmd\fR will look for the initialization file
```

Comment:
> dmd will look for the initialization file

Even for a relatively long-time user of dmd (e.g. who knows what `dmd.conf` is) this explanation is kind of meaningless. 

Some questions that many newcomers may have:

* What is the purpose of this "initialization file"? What does it "initialize"? Why is it used?
* "initialization file" - a better name would be "compiler configuration" file.
* What kind of initialization / configuration does it contain? What are the syntax rules?
* What will happen if it doesn't find it?
* What happens if there are multiple such files?

Reply:
The full phrase is

> dmd will look for the initialization file dmd.conf

...which I think is fairly clear, although it probably has room for improvement.

In any case, this sentence is exactly the same as it was before. Again, I can try rewriting it, but doing so would increase the scope of this PR, and may be better suited to a follow-up PR.